apps/hash-graph/lib/authorization/src/lib.rs

```diff
@@ -1,8 +1,78 @@
-#![feature(associated_type_bounds)]
-#![feature(async_fn_in_trait)]
-#![feature(impl_trait_in_assoc_type)]
-#![feature(lint_reasons)]
+#![feature(
+    associated_type_bounds,
+    async_fn_in_trait,
+    impl_trait_in_assoc_type,
+    lint_reasons,
+    return_position_impl_trait_in_trait
+)]
+
+use std::future::Future;
+
+use futures::Stream;
+use graph_types::{account::AccountId, knowledge::entity::EntityId};
+
+use crate::{
+    backend::CheckError,
+    zanzibar::{Consistency, Zookie},
+};
 
 pub mod backend;
 
 pub mod zanzibar;
+
+pub trait AuthorizationApi {
```

Comment:
(not strictly a review) just for anyone wondering. The earlier `AuthorizationApi` has been renamed into `ZanzibarBackend`, and a new `AuthorizationApi` for use in the REST-API has been created.